### PR TITLE
✨  Add Pubtech type for amp-consent

### PR DESCRIPTION
**Pull request by MProntera**
Adding Pubtech to the list of registered CMPs.

Added:
- Pubtech CMP config
- Example on cmp-vendors page
- Example documentation.

Pubtech CMP is a consent management provider (CMP) solution built on and registered with the IAB Europe Transparency and Consent framework.

Context: https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/integrating-consent.md

cc @ampproject/wg-monetization

### CODE REVIEW

File: extensions/amp-consent/amp-consent.md [hunk 1]
```diff
@@ -629,6 +629,7 @@ Join in on the discussion where we are discussing [upcoming potential features](
 -   Ogury : [Website](https://www.ogury.com/) - [Documentation](./cmps/ogury.md)
 -   OneTrust: [Website](https://www.onetrust.com/) - [Documentation](./cmps/onetrust.md)
 -   opencmp : [Documentation](./cmps/opencmp.md)
+-   Pubtech : [Documentation](./cmps/pubtech.md)
```
Comment (micajuine-ho) [inline extensions/amp-consent/amp-consent.md:632]: Can you add your website: https://www.pubtech.ai/

File: extensions/amp-consent/cmps/pubtech.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!---
+Copyright 2020 The AMP HTML Authors. All Rights Reserved.
```
Comment (micajuine-ho) [inline extensions/amp-consent/cmps/pubtech.md:2]: nit 2021

File: examples/amp-consent/cmp-vendors.amp.html [hunk 3]
```diff
@@ -320,6 +321,24 @@ <h3>Image that is NOT blocked by consent</h3>
...
+                "postPromptUI": "pubtech-post-prompt",
+                "clientConfig": {
+                  "privacyUrl": "",
+                  "isAmp": true,
+                  "websiteName": "Pubtech"
+                }
+              }
+            </script>
+        <div id="pubtech-post-prompt">
+          <button on="tap:consent.prompt(consent=pubtech)" role="button">Privacy settings</button>
```
Comment (micajuine-ho) [inline examples/amp-consent/cmp-vendors.amp.html:337]: <button on="tap:pubtech.prompt(consent=pubtech)" role="button">Privacy settings</button>

**Review by micajuine-ho — CHANGES_REQUESTED** [review]
Thank you for your PR. Left a few nits below.

Additionally, when I loaded the consent prompt on a local test page, I saw this: 
Screen Shot 2021-01-12 at 3 17 54 PM

File: extensions/amp-consent/amp-consent.md (hunk 1 above)
Comment (MProntera) [reply extensions/amp-consent/amp-consent.md]: Yes, thanks!

File: extensions/amp-consent/cmps/pubtech.md (hunk 2 above)
Comment (MProntera) [reply extensions/amp-consent/cmps/pubtech.md]: Right 👍

File: examples/amp-consent/cmp-vendors.amp.html (hunk 3 above)
Comment (MProntera) [reply examples/amp-consent/cmp-vendors.amp.html]: Done

**Comment by MProntera** [comment]
> Thank you for your PR. Left a few nits below.
> 
> Additionally, when I loaded the consent prompt on a local test page, I saw this:
> Screen Shot 2021-01-12 at 3 17 54 PM

@micajuine-ho 
I have changed the behavior of the user interface, unfortunately I have not found any guide that tells me how to do a test locally.
Can you tell me how to do it?
Otherwise if it's not a problem, I can test it once it's released.

**Comment by micajuine-ho** [comment]
This should help: https://github.com/ampproject/amphtml/blob/master/contributing/TESTING.md

On your branch, you can run 
```
gulp
```

and then test on any of the manual test pages (i.e. make changes to `examples/amp-consent/cmp-vendors.amp.html` to only run you amp-consent element and then test locally via localhost `http://localhost:8000/examples/amp-consent/cmp-vendors.amp.html`.

**Comment by MProntera** [comment]
> This should help: https://github.com/ampproject/amphtml/blob/master/contributing/TESTING.md
> 
> On your branch, you can run
> 
> ```
> gulp
> ```
> 
> and then test on any of the manual test pages (i.e. make changes to `examples/amp-consent/cmp-vendors.amp.html` to only run you amp-consent element and then test locally via localhost `http://localhost:8000/examples/amp-consent/cmp-vendors.amp.html`.

@micajuine-ho 
Thanks for your help!
I fixed the UI, can you check so let's proceed with the validation of the PR?

**Review by micajuine-ho — APPROVED** [review]
Everything looks good!

**Pull request merged**